src/app.js

```diff
@@ -197,11 +199,14 @@ export function app(app) {
     function removeChild() {
       parent.removeChild(element)
     }
+    recoverNode(node)
```

Comment:
^^ blows up on all the keyed tests, and some of the async tests...

I think the reusableNodes list will get recycled before they get a chance to be reused if I recycle up in render()...  which is what caused me to move the recoverNode down into the patch so I could work around the keyed interference...